vlasovsolver/cpu_acc_transform.cpp

```diff
@@ -146,5 +148,34 @@ Eigen::Transform<Real,3,Eigen::Affine> compute_acceleration_transformation(
       }
    }
 
+   // If a bulk velocity is being forced here, perform that last, after things were gyrated in the Hall frame
+   // If a cell is a remote L2 and was not caught in the loop over neighbours of L1 cells, compute its forcing here
+   // (same in sysboundary/ionosphere.cpp, regarding the time check:)
+   // TODO Make this a more elegant solution
+   // Now it's hacky as the counter is incremented in vlasiator.cpp
+   if( (((Parameters::t                > (SBC::Ionosphere::solveCount-1) * SBC::Ionosphere::couplingInterval)
+      && (Parameters::t-Parameters::dt < (SBC::Ionosphere::solveCount-1) * SBC::Ionosphere::couplingInterval)
+      && SBC::Ionosphere::couplingInterval > 0)
+      || SBC::Ionosphere::couplingInterval == 0)
```

Comment:
@markusbattarbee you noted this, it is a copy from the same mechanism in ionosphere.cpp, do we want to make this more elegant now? I think we have other cats to flog right now maybe?

Reply:
We do have many cats to flog, but if we don't flog this now, I fear it will remain unflogged for a long time, if not forever. We could do "bool globalflags::solveIonosphere = 0" in vlasiator.cpp and flip that with this test, no?

Reply:
[ec2f002](https://github.com/fmihpc/vlasiator/pull/664/commits/ec2f002601fbe5da0425e087713040cf6a71759d) is trying to fix this. Test in queue.